File: eLog.Domain/Models/LeavePolicySetting.cs
namespace eLog.Domain.Models
{
    public class LeavePolicySetting:BaseEntity
    {
        public int LeavePolicySettingID { get; set; }

        public int CompanyID { get; set; }

        public int LeaveTypeID { get; set; }

        public DateTime EffectiveDate { get; set; }

        public int InitialBalance { get; set; }

        public int EarnDays { get; set; }

        public int EarnPeriodID { get; set; }

        public String ResetAtID { get; set; }

        public int ResetDays { get; set; }

        public int MaxAvailableDays { get; set; }

        public bool EnableSandwich { get; set; }

        public bool DisableNegativeBalance { get; set; }

        public int SandwichCount { get; set; }

        public int? ModifiedBy { get; set; }

        public DateTime? ModifiedTS { get; set; }

        public bool IsActive { get; set; }

        public int LeavePolicyID { get; set; }
    }
}

File: eLog.Web/wwwroot/src/app/Models/leave/leavepolicysetting.ts
export interface leavepolicysetting
{
    leavePolicySettingID: number;
    companyID: number;
    leaveTypeID: number;
    effectiveDate: Date;
    initialBalance: number;
    earnDays: number;
    earnPeriodId: number;
    resetAtID: string;
    resetDay: number;
    maxAvailableDays: number;
    enableSandwich: boolean;
    disabaleNegativeBalance: boolean;
    sandwichCount: number;
    isActive: boolean;
    createdBy: number;
    createdTS: Date;
    leavePolicyID: number;
    description: string;
}
